src/Parser/Monk/Windwalker/Modules/Features/AlwaysBeCasting.js

```diff
@@ -7,6 +7,9 @@ import { formatPercentage } from 'common/format';
 import { STATISTIC_ORDER } from 'Main/StatisticBox';
 
 class AlwaysBeCasting extends CoreAlwaysBeCasting {
+    // Windwalker GCD is 1 second by default and static in almost all cases, 750 is lowest recorded GCD
+  static BASE_GCD = 1000;
+  static MINIMUM_GCD = 1000;
```

Comment:
I recommend looking at how I configured this recently for Brewmaster; there's no reason to configure a minimum equal to base if you're setting `STATIC_GCD_ABILITIES` and it will actually bug out spells that *are* affected by Haste.

Reply:
I did try it out with minimum GCD set to 750, but all abilities get hasted GCDs then, regardless if they're being set with `STATIC_GCD_ABILITIES` or not

Reply:
Ohh, it was working for brewmaster...